pkg/apis/pingcap/v1alpha1/backup.go

```diff
@@ -201,6 +201,22 @@ func IsBackupScheduled(backup *Backup) bool {
 	return condition != nil && condition.Status == corev1.ConditionTrue
 }
 
+// HaveTruncateUntil returns true if a Backup has truncate until set
+func HaveTruncateUntil(backup *Backup) bool {
+	if backup.Spec.Mode != BackupModeLog {
+		return false
+	}
+	return backup.Spec.LogTruncateUntil != ""
+}
+
+func IsBackupPaused(backup *Backup) bool {
+	if backup.Spec.Mode == BackupModeLog {
+		return IsLogBackupSubCommandOntheCondition(backup, BackupPaused)
+	}
+	_, condition := GetBackupCondition(&backup.Status, BackupPaused)
+	return condition != nil && condition.Status == corev1.ConditionTrue
```

Comment:
This only means the log backup is ever paused, not means it is paused. I guess you want to check if the log backup is paused. Only `backup.Status.Phase` represents the current status.

Reply:
deleted